### ISSUE TITLE
Cannot run SPC from docker

### ISSUE THREAD

**Issue by Gnafu** [GeoNode/geonode#4603]
Trying to start the SPC with docker-compose fails

## Expected Behavior
The docker-compose for SPC GeoNode should bring up all the systems needed for it to work

## Actual Behavior
A docker image is not found

```
Successfully built 88d4d52fadb3
Successfully tagged olivierdalang/pgdumper:latest
Pulling pgdumper (olivierdalang/spcgeonode:pgdumper-dev)...
ERROR: manifest for olivierdalang/spcgeonode:pgdumper-dev not found
```

## Steps to Reproduce the Problem

  1. clone the repository
  1. cd scripts/spcgeonode
  1. docker-compose up --build -d django geoserver postgres nginx (this works)
  1. docker-compose up --build -d

## Specifications

  - GeoNode version: master, commit dd16071bfbe720be0f1c435fe925abe101f6ddd2
  - Installation method: SPCGeoNode Docker
  - Platform: Ubuntu 18.04.2 LTS
  - Additional details: Followed these instructions http://docs.geonode.org/en/master/install/spc/index.html

**Comment by gannebamm** [comment]
this file is not longer present in olivierdalangs docker hub: olivierdalang/spcgeonode:pgdumper-dev

Therefore I changed the .override file to use -latest (#4611)

**Comment by afabiani** [comment]
Fixed by PR #4611
